wallet/wallet.go

```diff
@@ -35,7 +30,16 @@ func New(api API, maddr *address.Address, iam big.Int) (*Module, error) {
 
 // NewAddress creates a new wallet
 func (m *Module) NewAddress(ctx context.Context, typ string) (string, error) {
-	addr, err := m.api.WalletNew(ctx, typ)
+	ty := crypto.SigTypeUnknown
+	if "bls" == typ {
+		ty = crypto.SigTypeBLS
+	} else if "secp256k1" == typ {
+		ty = crypto.SigTypeSecp256k1
+	} else {
+		return "", fmt.Errorf("unkown wallet type %s", typ)
+	}
```

Comment:
A quick fix for the `testnet/3` change since now it's typed, not a string.
I won't imagine the types of wallet to change much, so isn't that bad... but don't fully like this kind of ifs.